panther_lights/include/panther_lights/lights_controller_node.hpp

```diff
@@ -42,7 +42,7 @@ class ControllerNode : public rclcpp::Node
 {
 public:
   ControllerNode(
-    const std::string & node_name, const rclcpp::NodeOptions & options = rclcpp::NodeOptions());
+    const rclcpp::NodeOptions & options = rclcpp::NodeOptions().use_intra_process_comms(true));
```

Comment:
It'd be better not to hardcode the intra-process communication here. We can use arguments in the launch file to specify this configuration.